src/Templates/Microsoft.DotNetCore.CSharp.ProjectTemplates.Common/ClassLibrary-CSharp-NetCoreApp.vstemplate

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
+  <TemplateData>
+    <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
+    <Icon Package="{FAE04EC1-301F-11d3-BF4B-00C04F79EFBC}" ID="4547" />
+    <ProjectType>CSharp</ProjectType>
+    <RequiredFrameworkVersion>2.0</RequiredFrameworkVersion>
+    <SortOrder>14</SortOrder>
+    <NumberOfParentCategoriesToRollUp>0</NumberOfParentCategoriesToRollUp>
+    <TemplateID>Microsoft.CSharp.NETCore.ClassLibrary</TemplateID>
+    <CreateNewFolder>true</CreateNewFolder>
+    <CreateInPlace>true</CreateInPlace>
+    <DefaultName>ClassLibrary</DefaultName>
+    <ProvideDefaultName>true</ProvideDefaultName>
+    <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
+  </TemplateData>
+  <TemplateContent>
+    <ProjectCollection/>
+    <CustomParameters>
+      <CustomParameter Name="$language$" Value="C#" />
+      <CustomParameter Name="$uistyle$" Value="none" />
+      <CustomParameter Name="groupid" Value="Microsoft.Common.Library" />
```

Comment:
This will create a netstandard library and not a library with a netcoreapp TFM. Today we have two class library projects - one under the .NET Core node targeting netcoreapp1.1 and another under the NETStandard node. (https://github.com/dotnet/project-system/issues/1909)

You could pass in TargetFrameworkOverride here but then I assume that gets in the way of lightup. You probably need to have another template added to Dotnet/templating? @mlorbetske